### ISSUE TITLE
Pregenerated serializer hangs on multidimensional array context creation

### ISSUE THREAD

**Issue by internetcoffeephone** [msgpack/msgpack-cli#81]
I'm using Unity 5, so I pregenerated the serializers, and now I'm creating the right context. There are no circular references, and I'm adding the serializers to the context in the right order (if that matters).
It hangs there however, to be specific in this part:

```
public XxxxSerializer(MsgPack.Serialization.SerializationContext context) : 
                base(context) {
            this._serializer0 = context.GetSerializer<Yyyy[,,]>();
        }
```

Somewhere down the stack trace it seems to be hanging at:

```
MsgPack.ReflectionAbstractions.GetIsGenericType (source={Yyyy[,,]})
```

**Comment by yfakariya** [comment]
Thank you for reporting and sorry for delay.
I have never considered multidimensional array, so it is missing feature.

**Comment by yfakariya** [comment]
Fixed in 0.6.0-beta5 with 1-dimensional array which has non-zero lower-bound.
